textproc/pdfjam/Portfile

```diff
@@ -34,23 +34,20 @@ post-patch {
     reinplace "s|/usr/local|${prefix}|g" \
         ${worksrcpath}/bin/pdfjam \
         ${worksrcpath}/man1/pdfjam.1 \
-        ${worksrcpath}/PDFjam-README.html
+        ${worksrcpath}/README.md
 }
 
 build {}
 
 destroot {
-    delete ${destroot}${prefix}/bin ${destroot}${prefix}/share/man/man1
-    copy ${worksrcpath}/bin ${destroot}${prefix}/bin
-    copy ${worksrcpath}/man1 ${destroot}${prefix}/share/man/man1
+    xinstall -m 755 ${worksrcpath}/bin/pdfjam ${destroot}${prefix}/bin/pdfjam
+
+    #xinstall -m 755 -d ${destroot}${prefix}/man/man1
```

Comment:
Delete the line if not needed?

Reply:
@breun Could we do that and then we can merge this?

Reply:
I've updated the PR to remove the commented line.

I don't know if it's safe to merge, because I don't know if anything (or anyone) depends on the wrapper scripts that were removed in version 3.02, like I mentioned in the description of this PR. I was hoping the maintainer of this port could be the judge of that, and decide whether a port needs to be added for the (now unmaintained, but working) new pdfjam-extras repository.

Reply:
The maintainer no longer has a Mac, so I think you are the closest person to a real maintainer we have now. @jjstickel Would you mind giving up the maintainership?

Reply:
Actually, I got a Mac again a week ago! Still getting setup. I have no concerns about someone else taking maintainership of this port. It should be `openmaintainer`in any case.

Reply:
I've created a pull request for a separate `pdfjam-extras` port for the scripts that were split off from `pdfjam` to the [pdfjam-extras repository](https://github.com/rrthomas/pdfjam-extras): https://github.com/macports/macports-ports/pull/20201

Reply:
Actually, I got a Mac again a week ago! Still getting setup. I have no concerns about someone else taking maintainership of this port. It should be openmaintainer in any case.